cfgov/jinja2/v1/wagtail-demo/event/index.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block title -%}
+    Consumer Financial Protection Bureau
+{%- endblock %}
+
+{% block desc -%}
+    Our vision is a consumer finance marketplace that works for American consumers,
+    responsible providers, and the economy as a whole.
+{%- endblock %}
+
+{% block content_main %}
+    <div class="block
+                block__flush-top">
+        {{ page.body|richtext }}
+    </div>
+
+    <div class="content-l">
+        <div class="content-l_col-1-2">
+            {% for item in page.agenda_items %}
+                {{ render_stream_child(item) }}
+            {% endfor %}
```

Comment:
nope, doesn't work. renders some elements but any richtext fields that are child get rendered as raw html, so had to decode it on the back end